app/controllers/sitemaps_controller.rb

```diff
@@ -1,8 +1,10 @@
 class SitemapsController < ApplicationController
   def show
     @courses = Course
-      .includes(:provider)
       .where(recruitment_cycle_year: Settings.current_cycle)
+      .select("course_code", "provider_code", "changed_at")
+      .page(1)
+      .per(20000)
```

Comment:
What if there's more than 20k. Could this value be dynamic like `Course.count()` or something so its also return all the results on one page

Reply:
The backend is hardcoded to 20K. So if the number is over that it'll chop it off (not sure but might go to next page?).